packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -47,6 +48,10 @@ type AuthDataMap = Record<
     }
 >
 
+type AuthData = ShopperLoginTypes.TokenResponse & {
+    customer_type: string
+}
```

Comment:
could you add a comment explain why we need this type (why extended from TokenResponse)? 

Explain it that this is not from the API response, we manually store the auth type, so we don't need to make another API call when we already have the data. Plus, the getCustomer endpoint it's weird, guest user returns 404, and it's a pain to deal with this in the user land.

Reply:
> Code Tells You How, Comments Tell You Why

Reply:
Gotcha, the comment was added. Thanks for the feedback